src/sentry/api/serializers/models/exporteddata.py

```diff
@@ -29,6 +29,7 @@ def serialize(self, obj, attrs, user, **kwargs):
             "dateCreated": obj.date_added,
             "dateFinished": obj.date_finished,
             "dateExpired": obj.date_expired,
+            "dateExpiredString": obj.date_expired_string,
```

Comment:
Probably include this in a separate pr since it's unrelated

Reply:
Didn't realize i'd staged that -- removed